_posts/2021-01-13-hapi-fhir.md

```diff
@@ -235,19 +247,13 @@ import java.util.List;
 import java.util.Map;
 
 @Component
-public class PatientProvidor implements IResourceProvider {
+public class PatientProvider implements IResourceProvider {
 
     @Autowired
     private FhirContext ctx;
 
     @Autowired
     private IPatient patientDao;
```

Comment:
The naming "IPatient" suggest that this is an interface for a Patient implementation, but in your code it's actually an interface for the DAO. I would just rename "IPatient" to "PatientDAO".